src/test/java/be/syntra/devshop/DevshopFront/controllers/SearchControllerTest.java

```diff
@@ -25,11 +22,10 @@
 import org.springframework.test.web.servlet.ResultActions;
 
 import java.math.BigDecimal;
-import java.util.List;
```

Comment:
for this whole class: 
if a String isn't static final -> no allCaps
remove comments between the code
there were parameterized tests here, no reason to splits them up. So please set them back

Reply:
combined price and name filtering into parameterized test and added another parameterized test for archived sorting